impress-org/givewp
# Add support for PayPal Order ID via gateway data (Next Gen)

## Description



This PR adds support for passing the PayPal Order ID via the Gateway Data.

Previously, the Order ID was pulled directly from the `$_POST` global, which resulted in a `PayPalOrderIdException` for a missing Order ID in the Next Gen PayPal Commerce gateway implementation.

## Testing Instructions



Make a donation using the Next Gen updated PayPal Commerce gateway.

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1204267265032333
